### PR TITLE
feat: add wishlist page

### PR DESCRIPTION
**Pull request by anik31**
Wishlist page for Ecommerce project.
wishlist.html contains the HTML code & style.css contains the CSS code.

Deploy preview - https://deploy-preview-2--vacay-store.netlify.app/wishlist

### CODE REVIEW

**Review by foolhardy21 — APPROVED** [review]
LGTM. one suggestion - you can change grid template area in media query for responsiveness.

**Comment by anik31** [comment]
> LGTM. one suggestion - you can change grid template area in media query for responsiveness.

Thanks for the suggestion @foolhardy21 . I will be adding responsiveness after completing the assignments.

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="search-box">
+            <i class="fas fa-search"></i>
+            <input type="text" placeholder="Search">
+        </div>
+        <nav class="navigation">
+            <ul>
+                <li><a href="#" class="btn btn-primary">Login</a></li>
+                <li>
+                    <div class="badge-wrapper">
+                        <a href="#"><i class="btn-icon far fa-heart"></i></a>
```
Comment (vrx29) [inline wishlist.html]: extra div can be removed. you have a wrapper of li can give the class="badge-wrapper" to li directly. Check if it works.

Comment (anik31) [reply wishlist.html]: Yes it works @vrx29 . I have removed the extra div. Thanks

**Pull request merged**